Makefile

```diff
@@ -279,6 +279,7 @@ $(STATUS_CLIENT_APPIMAGE): nim_status_client $(APPIMAGE_TOOL) nim-status.desktop
 
 	mkdir -p pkg
 	$(APPIMAGE_TOOL) tmp/linux/dist $(STATUS_CLIENT_APPIMAGE)
+	cd pkg && tar czvf $(shell basename $(STATUS_CLIENT_APPIMAGE)).tar.gz $(shell basename $(STATUS_CLIENT_APPIMAGE))
```

Comment:
Are we sure we want to make this `make` target EVEN BIGGER? It's already far too big.

Considering this is only useful for uploaded releases that people download, what's the point of doing this every time the `make` target is used? Wouldn't it make more sense to tarball it only in CI builds for upload?

Reply:
I did it like that so I could easily test that it worked as intended on my Linux laptop, and then went ahead with it (plus Jenkinsfile changes) for the PR. But as you suggest, we could instead do it in a step that only runs in CI. If we go with that approach, where is the correct place to introduce the behavior?

Reply:
Either in the `Jenkinsfile` itself, or in the [`status-jenkins-lib`](https://github.com/status-im/status-jenkins-lib). But I can do that.

Reply:
What do you think about this? https://github.com/status-im/status-desktop/commit/601a92bd

Middle of the road. Not in the main target, but still in the `Makefile`.

Reply:
Not sure if you want to include the `.AppImage` part in the `.tar.gz` filename.

Reply:
> Not sure if you want to include the `.AppImage` part in the `.tar.gz` filename.

We can just make it `tar.gz`, leave out the `.AppImage`. I think it will be pretty obvious to a Linux user that "the tar.gz is for for me".